modules/persistent-storage-csi-azure-disk-perf-plus-from-snapshots.adoc

```diff
@@ -0,0 +1,105 @@
+//
```

Comment:
I think we can remove the file as its content isn't used or referenced.

Reply:
Done.